src/Action.php

```diff
@@ -70,7 +70,7 @@ protected function handle(Actionable $action)
         try {
             $response = $action();
         } catch (Throwable $throwable) {
-            $this->handleFailure($action, $throwable);
+            $action->failed($throwable);
```

Comment:
Since we're removing the `failed()` method from the Stub and we're not checking for the method existence, wouldn't this give an error if the user doesn't add one?